src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts

```diff
@@ -131,4 +131,8 @@ suite('Local Content Manager', function (): void {
 		let displayOutput = <nb.IDisplayData>notebook.cells[0].outputs[0];
 		assert.equal(displayOutput.data['text/html'], '<div></div>');
 	});
+	test('Should create a new empty notebook if content is undefined', async function (): Promise<void> {
+		let content = await contentManager.loadFromContentString('');
+		assert.equal(content.cells.length, 0);
```

Comment:
Since there's only the one assert here it's not strictly necessary, but it'd be good to get in the habit of always including a descriptive message for these so when they fail it's easy to tell what went wrong without needing to look at the code.